util/sliceUtil/SliceUtil.go

```diff
@@ -0,0 +1,57 @@
+package sliceUtil
```

Comment:
we need to add test cases for these functions

Reply:
these functions seems useful as we do these operations at multiple places. we will start using these . so proper test cases are required

Reply:
added